app/controllers/reviews_controller.rb

```diff
@@ -30,9 +30,9 @@ def edit
 
   def update
     @trail = Trail.find(params[:trail_id])
-    @review = Review.find(params[:id])
-    if  current_user.id == @review.user_id || current_user.admin?
-      if @review.update(review_params)
+    review = Review.find(params[:id])
+    if  current_user.id == review.user_id || current_user.admin?
```

Comment:
Since you'll have to update the below, please also remove this extra white space between `if` and `current_user`